bgpd/bgp_routemap.c

```diff
@@ -5897,10 +5897,11 @@ DEFUN_YANG (set_table_id,
 
 DEFUN_YANG (no_set_table_id,
 	    no_set_table_id_cmd,
-	    "no set table",
```

Comment:
what does the option id field do? It is not parsed so is it just for the cli to accept no form?

Reply:
yep